core/src/main/java/com/sequenceiq/cloudbreak/service/externaldatabase/AzureDatabaseServerParameterSetter.java

```diff
@@ -0,0 +1,32 @@
+package com.sequenceiq.cloudbreak.service.externaldatabase;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.common.mappable.CloudPlatform;
+import com.sequenceiq.cloudbreak.service.externaldatabase.model.DatabaseServerParameter;
+import com.sequenceiq.redbeams.api.endpoint.v4.stacks.DatabaseServerV4StackRequest;
+import com.sequenceiq.redbeams.api.endpoint.v4.stacks.azure.AzureDatabaseServerV4Parameters;
+
+@Component
+public class AzureDatabaseServerParameterSetter implements DatabaseServerParameterSetter {
+
+    @Value("${cb.azure.externaldatabase.retentionperiod:7}")
```

Comment:
I think the same is used in sdx app so we should expose these classes into a common modul. What do you think ?